pkgs/development/compilers/haxe/default.nix

```diff
@@ -31,7 +31,7 @@ let
       inherit version;
 
       buildInputs = [ zlib pcre neko ]
-        ++ lib.optional (lib.versionAtLeast version "4.1") [ mbedtls ]
+        ++ lib.optionals (lib.versionAtLeast version "4.1") ([ mbedtls ] ++ lib.optional stdenv.hostPlatform.isDarwin [ Security ])
```

Comment:
I think this is a bit more readable this way. Also: Are you sure this issue doesn't affect 4.0 or any previous version?

```suggestion
        ++ lib.optionals (lib.versionAtLeast version "4.1") [ mbedtls ]
        ++ lib.optionals (lib.versionAtLeast version "4.1" && stdenv.hostPlatform.isDarwin) [ Security ]
```

Also using just `stdenv.isDarwin` is fine.

Reply:
> Also: Are you sure this issue doesn't affect 4.0 or any previous version?

Tested all packaged versions. 3.x and 4.0 compile fine, only 4.1 and 4.2 error out.

![Bildschirmfoto von 2021-04-20 22-11-11](https://user-images.githubusercontent.com/23431373/115457940-63fdd000-a225-11eb-9cff-ef877ca7f27f.png)